packages/ag-charts-community/src/chart/interaction/toolbarManager.ts

```diff
@@ -126,7 +126,7 @@ export class ToolbarManager extends BaseManager<EventTypes, ToolbarEvent> {
     updateButton<T extends ToolbarGroup>(
         group: T,
         id: string,
-        options: { label?: string; icon?: AgIconName; fill?: string }
+        options: { label?: string; icon?: AgIconName; fill?: string; strokeWidth?: number }
```

Comment:
I think if we do this a third time, beyond fill and stroke width, it would be sensible to create some kind of custom renderer function option instead and moving these behaviours to the annotations module. Not for now though!